src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @title WET-BOEW Hello world plugin
+ * @overview Plugin contained to show an example of how to create your custom WET plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+
+	/**
+	 * Initiate chat wizard bubble
+	 * @method initiateBubble
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	initiateBubble = function( $selector ) {
```

Comment:
The function name need to be more descriptive because it is confusing with the other `init` functions. Both are initiatlizing the bubble. You can create multiple sub function with a descriptive name (this include also the code in the init function).